### PR TITLE
JSONify Error Response

### PR DESCRIPTION
**Pull request by kdiluca**

### CODE REVIEW

File: src/tyr/service.cc [hunk 1]
```diff
@@ -795,18 +795,42 @@ namespace {
       stream << *json;
     }
   }
+}
+
+const headers_t::value_type CORS{"Access-Control-Allow-Origin", "*"};
```
Comment (kevinkreiser) [inline src/tyr/service.cc]: these can go in the anonymous namespace

File: src/tyr/service.cc [hunk 2]
```diff
@@ -795,18 +795,42 @@ namespace {
...
+const headers_t::value_type JSON_MIME{"Content-type", "application/json;charset=utf-8"};
+const headers_t::value_type JS_MIME{"Content-type", "application/javascript;charset=utf-8"};
+
+namespace valhalla {
+  namespace tyr {
 
-  const headers_t::value_type CORS{"Access-Control-Allow-Origin", "*"};
-  const headers_t::value_type JSON_MIME{"Content-type", "application/json;charset=utf-8"};
-  const headers_t::value_type JS_MIME{"Content-type", "application/javascript;charset=utf-8"};
+    worker_t::result_t tyr_worker_t::jsonify_error(uint64_t code, const std::string& status, const std::string& error, http_request_t::info_t& request_info) const {
```
Comment (kevinkreiser) [inline src/tyr/service.cc]: this can go in the anonymous namespace

File: valhalla/tyr/service.h [hunk 3]
```diff
@@ -3,11 +3,31 @@
 
 #include <boost/property_tree/ptree.hpp>
 
+#include <prime_server/prime_server.hpp>
+#include <prime_server/http_protocol.hpp>
+
+#include <valhalla/baldr/json.h>
```
Comment (kevinkreiser) [inline valhalla/tyr/service.h]: dont need this one in here, move to src

File: src/tyr/service.cc (hunk 2 above)
Comment (kdiluca) [inline src/tyr/service.cc]: These are fixed now!

**Comment by kevinkreiser** [comment]
:rocket:

**Pull request merged**